main.js

```diff
@@ -919,7 +919,7 @@ function extendChannel(nodeID, comClass, valueId) {
             if (role && !stateObj.common.role) stateObj.common.role = role;
 
             stateObj.native = valueId;
-            adapter.log.info('State updated: ' + stateID + ' = ' + value + ', index = ' + valueId.index + ', comClass = ' + comClass + ', instance = ' + valueId.instance);
+            adapter.log.debug('State updated: ' + stateID + ' = ' + value + ', index = ' + valueId.index + ', comClass = ' + comClass + ', instance = ' + valueId.instance);
```

Comment:
What's the logic behind these loglevel changes? Some went from info to debug, some the other way round?

Reply:
I had a look at the log statements to verify what is actually necessary. For example state changes are normal behavior and clutter the log file. But if functions are called which are not implemented (like polling) that should be visible.